terraform/environments/electronic-monitoring-data/application_variables.json

```diff
@@ -1,16 +1,57 @@
 {
   "accounts": {
     "development": {
-      "example_var": "dev-data"
+      "example_var": "dev-data",
+      "enable_dbt_k8s_secrets": true,
+      "analytical_platform_share": [
+        {
+          "target_account_name": "analytical-platform-data-production",
+          "target_account_id": "593291632749",
+          "assume_account_name": "analytical-platform-management-production",
+          "assume_account_id": "042130406152",
+          "data_locations": [
+            "dpr-structured-historical-production"
```

Comment:
don't think that's right

Reply:
oh also, that account id just above?

Reply:
yeah i tried to find out what data locations meant in terms of lake formation and the answers werent totally clear.

not sure what the `analytical-platform-management-production` account is but sounds like it might be needed

Reply:
https://github.com/ministryofjustice/modernisation-platform-environments/blob/b2602a02727f1e12ade91125dde9e433e74c3ce3/terraform/environments/analytical-platform-compute/iam-roles.tf#L235